packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/FGMClusterManagersController.h

```diff
@@ -0,0 +1,70 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <Flutter/Flutter.h>
+#import <GoogleMaps/GoogleMaps.h>
+@import GoogleMapsUtils;
+
+#import "messages.g.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+// Defines cluster managers controller interface which
+// is responsible for adding/removing/returning one or more cluster managers.
+@interface FGMClusterManagersController : NSObject
+
+/// Initializes FLTClusterManagersController.
+///
+/// @param callbackHandler A callback handler.
+/// @param mapView A map view that will be used to display clustered markers.
+- (instancetype)initWithMapView:(GMSMapView *)mapView
+                callbackHandler:(FGMMapsCallbackApi *)callbackHandler;
+
+/// Creates ClusterManagers and initializes them form JSON data.
```

Comment:
You've used `ClusterManager` throughout these comments, but that's not an actual type. In general, you should not use type names in declaration comments unless they resolve some ambiguity (e.g., in the conversion function comments, where the point is that it's two different concrete class representations of the same conceptual thing); you should use English. If you do need to use a type name to resolve ambiguity, it needs to be an exact type name.